CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ All notable changes to `webhook` will be documented in this file
 ## 2.1.0 - 2020-08-21
 
 * Added Laravel 8 support.
+* Dropped Laravel 6 support.
+* Dropped PHP7.2 support.
```

Comment:
Laravel 7 supports PHP >= 7.2.5. I dont think we can drop support for it now.

Reply:
Fixed !